lightwood/api/encode.py

```diff
@@ -4,11 +4,11 @@
 from lightwood.data.encoded_ds import EncodedDs
 
 
-def encode(encoders: List[BaseEncoder], folds: List[pd.DataFrame], target: str) -> List[EncodedDs]:
-    if isinstance(folds, pd.DataFrame):
-        folds = [folds]
+def encode(encoders: List[BaseEncoder], subsets: List[pd.DataFrame], target: str) -> List[EncodedDs]:
+    if isinstance(subsets, pd.DataFrame):
+        subsets = [subsets]
 
     encoded_ds_arr: List[EncodedDs] = []
-    for fold in folds:
+    for fold in subsets:
```

Comment:
maybe nice to call this "s" instead of fold but this is also fine.

Reply:
should be `subset` will rename, not sure why it's this way rn